theme/src/components/App/App.module.scss

```diff
@@ -59,18 +59,69 @@
       @apply relative p-4;
     }
 
+    // Styles for code copy button.
     .copybtn {
       @apply absolute top-4 right-4 bg-white;
+      // Render invisible by default.
       @apply opacity-0 transition-opacity;
-    }
 
-    .copybtn img {
-      @apply m-0 w-6 h-6;
+      // Fix dimensions of copy button image.
+      img {
+        @apply m-0 w-6 h-6;
+      }
     }
 
+    // Animate copy button into view by changing opacity to 100%.
     .highlight:hover .copybtn {
       @apply opacity-100;
     }
+
+    // Page content sections.
```

Comment:
could you add a clarifying comment to specify that these are overriding jupyterbook defaults and why we need to do so?